i18n/zh/docusaurus-plugin-content-docs/current/user-guide/deployments-administration/maintenance/recovery-mode.md

```diff
@@ -0,0 +1,59 @@
+---
+keywords: [GreptimeDB, 恢复模式, metasrv, 集群管理, 数据恢复]
+description: 介绍如何使用 GreptimeDB 集群恢复模式来解决 Datanode 启动失败问题，并从 region 数据丢失或损坏中恢复。
+---
+
+# 集群恢复模式
+
+恢复模式是 GreptimeDB 中的一个安全特性，允许用户手动恢复集群从失败状态。
```

Comment:
```suggestion
恢复模式是 GreptimeDB 中的一个安全特性，允许开发者手动恢复集群从失败状态。
```

Reply:
resolved in[f543376](https://github.com/GreptimeTeam/docs/pull/2013/commits/f5433761bb8e74f494210ba766397ed863bec0e9)